src/app/projects/project-list/project-list.component.ts

```diff
@@ -97,4 +104,15 @@ export class ProjectListComponent implements OnInit {
       this.initFavoriteProjects();
     });
   }
+
+  private updateTitle() {
+    switch (this.activeTabIndex) {
+      case 0:
+        return this.titleService.setTitle('All projects - Wharf');
+      case 1:
+        return this.titleService.setTitle('Favorite projects - Wharf');
+      default:
+        return this.titleService.setTitle('Wharf');
+    }
+  }
```

Comment:
Suggest removing the return statements. I find them confusing, as the setTitle doesn't return anything. Placing them after the switch statements would be semantically equivalent.

And that third case is superfluous. It will never be used, so suggest using a simple `if` statement instead

```suggestion
  private updateTitle() {
    if (this.activeTabIndex === 1) {
      this.titleService.setTitle('Favorite projects - Wharf');
    } else {
      this.titleService.setTitle('All projects - Wharf');
    }
  }
```